accord-core/src/main/java/accord/local/Node.java

```diff
@@ -558,6 +558,9 @@ protected void start(BiConsumer<? super R, Throwable> callback)
 
     public void reply(Id replyingToNode, ReplyContext replyContext, Reply send, Throwable failure)
     {
+        if (replyingToNode == Id.NONE)
```

Comment:
when is this case in the protocol?

Reply:
This _used_ to be the case during replay. But you are right, maybe with my latest changes this will not be the case anymore.

Reply:
Removed now.